packages/core/src/CorePlugin.ts

```diff
@@ -136,4 +142,7 @@ export class CorePlugin extends JWPlugin {
             direction: params.vSelection.direction,
         });
     }
+    applyFormat(): void {
+        return;
```

Comment:
I think the `Inline` plugin should define a `toggleFormat` command rather than `applyFormat`. It would look in the formats of the node to check if the format given in argument is already there, if yes remove them, if no, add it. (This would even allow more complex decisions for later on like the al's cases where we might detect that the formats that are already there have classes on them so we can't remove them so we need to toggle them off in a different way)